CHANGELOG.md

```diff
@@ -1,28 +1,35 @@
-# PWA Studio Release 12.6.0
+# PWA Studio Release 12.7.0
 
 **NOTE:**
-_This changelog only contains release notes for PWA Studio and Venia 12.6.0_.
+_This changelog only contains release notes for PWA Studio and Venia 12.7.0_.
 _For older release notes, see_ [PWA Studio releases][].
 
 ## Highlights
 
-The 12.6.0 release of PWA Studio focuses on accessibility improvements and bug fixes.
-  
--   Accessibility Improvements: Added autocomplete attributes to checkout shipping and payment forms — [3785][]  
--   Accessibility Improvements:: Open panels now have keyboard focus. — [3805][]   
--   Accessibility Improvements:: Navigation buttons for carousels now meet contrast requirements. — [3787][]  
--   Controls now have a descriptive accessible name. Visible \[label] elements are correctly associated with their inputs using the "for" attribute. — [3786][]  
--   Removes duplicate logic from QuoteGraphQlPwa module — [29][]  
--   Bug Fix: Failing cypress tests fixed — [3911][]  
--   Bug Fix - Braintree payment form Customer Name field values now appear normal. — [3912][]  
--   Bug Fix: Braintree npm package braintree-web-drop-in updated to latest version to fix csp related console errors. — [3912][]  
--   Accessibility Improvements: Keyboard focus no longer lands on hidden elements in Kebab menu — [3800][]   
--   Fixed a bug in mobile view where the Edit and Remove options in the kebab menu for a cart item on the cart page is non functional. - [3925][]
--   Fixed a console error when accessing different filters. - [30][]
+The 12.7.0 release of PWA Studio focuses on accessibility improvements and bug fixes.
 
-## 12.6.0 Lighthouse scores
+### Features
 
-With each new release of PWA Studio, we perform Lighthouse audits of four Venia page types, each representing a different level of complexity. Shown below are the Lighthouse scores for the 12.6.0 release of these pages on desktop and mobile devices.
+- **Out-of-stock for product variations** [3903][] — The out-of-stock setting can now be applied to configurable product variations, making it easier for customers to select variations.
```

Comment:
Should this [pr](https://github.com/magento/pwa-studio/pull/3903/) be part of Highlights as well? It was a pretty big pr with good improvements.

Reply:
Yeah, for sure. I missed that one for some reason. I'll add it in first thing in the morning. 👍

Reply:
@dpatil-magento, After following your pr link, it looks like the same thing as the feature mentioned: **Out-of-stock for product variations**. Is there another related feature that I'm missing?

Reply:
Never mind. I get it. Fixed.